constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -231,6 +245,72 @@ export class ClassEditPage extends React.Component {
                         onChange={(e) => this.handleChange(e, "inputClassKey")}
                     />
 
+                    <InputText
+                        label="Google Classroom Code"
+                        isTextBox={true}
+                        required={false}
+                        description="Enter the google classroom code"
+                        value={this.state.googleClassCode}
+                        onChangeCallback={(e) =>
+                            this.handleChange(e, "googleClassCode")
+                        }
+                    />
+
+                    <InputText
+                        label="Price Lump"
+                        isTextBox={true}
+                        required={false}
+                        description="Enter price for one time payment (Either enter only in this field or only in the price per session field)"
+                        onChangeCallback={(e) =>
+                            this.handleIntegerChange(e, "priceLump")
+                        }
+                        validators={[
+                            {
+                                validate: (input) => parseInt(input) != NaN,
+                                message: "price must be a valid number",
```

Comment:
I think all of these fields should be outside the the if clause (!isEdit). And should be below the fullState section. We may want to adjust prices or the google classroom code later